build.sh

```diff
@@ -801,6 +801,39 @@ case "$COMMAND" in
         ;;
 
     ######################################
+    # Continuous Integration
+    ######################################
+
+    "ci-pr")
+        mkdir -p build/reports
+
+        if [ "$target" = "docs" ]; then
+            ./build.sh set-swift-version
+            ./build.sh verify-docs
+        elif [ "$target" = "swiftlint" ]; then
+            ./build.sh verify-swiftlint
+        else
+            sha=$ghprbSourceBranch
```

Comment:
The `sha` env variable is used by the CocoaPods installation examples to target the appropriate branch. #2808 will also use `sha` for Carthage examples.